ansible/README.md

```diff
@@ -0,0 +1,59 @@
+# the ansible stuff
+
+## example `ansible/inventory.yml`
+you have to create a `ansible/inventory.yml` before you can run the playbook
+```yml
+---
+local:
+  hosts:
+    localhost:
+      ansible_connection: local
+      pipo_user:
+        from:
+          hostname: localhost
+          user: example
+        to:
+          user: pipo
+
+server_pipo:
+  hosts:
+    "mygrateappsrvc.xyz":
+      pipo_repository: https://github.com/Sife-ops/pipo # optional
+      pipo_version: "feature/ansible"                   # optional
+      
+server_synapse:
+  hosts:
+    "matrix.example.xyz":
+      app_service_pipo:
+        server_name: matrix.example.xyz
+        container_name: synapse_matrix.example.xyz      # optional
+        data_path: /root/synapse_matrix.example.xyz
+        registration:
```

Comment:
How does this get loaded into PIPO?

Reply:
thas not for pipo 2 load.
its not hardcoded bc its an ignored file.
those are parameters for the register_app_service role 2 register the pipo app service on a synapse server